src/utilities/__tests__/coerceValue-test.js

```diff
@@ -283,4 +284,39 @@ describe('coerceValue', () => {
       ]);
     });
   });
+
+  describe('for nested GraphQLList', () => {
+    const TestList = GraphQLList(GraphQLList(GraphQLInt));
+
+    it('correctly coerces a nested list value', () => {
```

Comment:
I think it's better to use existing naming convention, e.g.
https://github.com/graphql/graphql-js/blob/7bc632a22c358af7746e23231fa657a2adb3c25d/src/utilities/__tests__/coerceValue-test.js#L131-L134
https://github.com/graphql/graphql-js/blob/7bc632a22c358af7746e23231fa657a2adb3c25d/src/utilities/__tests__/coerceValue-test.js#L75-L78

So instead of `correctly coerces a nested list value` it could be `return value for nested list`.
Same goes for other testcases